Runtime/Helpers/Math.cs

```diff
@@ -35,5 +35,13 @@ float second_value
                 || ( first_value < 0.0f && second_value < 0.0f )
                 || ( first_value == 0.0f && second_value == 0.0f );
         }
+
+        public static int MathMod(
```

Comment:
I propose to add a note for this method, with some samples of the expected results to avoid confusion ( as modulo could have multiple behavior )